ush/job_preamble.sh

```diff
@@ -104,9 +114,28 @@ export -f POST_STEP
 if [ "${RUN_ENVIR}" = "nco" ]; then
     export COMIN="${COMIN_BASEDIR}/${RUN}.${PDY}/${cyc}"
     export COMOUT="${COMOUT_BASEDIR}/${RUN}.${PDY}/${cyc}"
+    export COMINaws="${AWSROOT}/${RUN}.${PDY}/${cyc}"
```

Comment:
I'm curious what AWSROOT is? Is it already documented somewhere? Is this AWS different from Amazon Web Services?

Reply:
> I'm curious what AWSROOT is? Is it already documented somewhere? Is this AWS different from Amazon Web Services?

It is being used as a holding spot for input files being brought into the system.  Has to do with some of the discussion/chatter recently about the GFS files not belonging in RRFS com space. 
Maybe COMINexternal would be more general?

Reply:
I will rename them to COMINext, EXTROOT and OPSROOT/ext if that is Ok? When we use wcoss2 sources COMINgfs, COMINhrrr etc, EXTROOT will just symlink the files and is used as a place to communicate the ICS/LBCS data to the `make_ics/lbcs`.

Reply:
That sounds good to me, @danielabdi-noaa

Reply:
I think that is a nicer alternative given that we would certainly have external sources that are not aws. Thanks @danielabdi-noaa and @MatthewPyle-NOAA